browser-test/src/phone.test.ts

```diff
@@ -137,14 +137,28 @@ describe('phone question for applicant flow', () => {
         'This phone number is invalid',
       )
     })
-    it('invalid characters in phone numbers', async () => {
+    it('invalid length of phone numbers', async () => {
       const {page, applicantQuestions} = ctx
       await applicantQuestions.applyProgram(programName)
       await applicantQuestions.answerPhoneQuestion(
         'United States',
         '123###1212',
       )
 
+      await applicantQuestions.clickNext()
+      const countryCodeId = '.cf-phone-number'
+      expect(await page.innerText(countryCodeId)).toContain(
+        'The phone number must contain only ten digits',
+      )
+    })
```

Comment:
Nitpick (non-blocking): Can we keep any empty line between each test case? (It looks like this file currently doesn't have empty lines between each case, so if you'd rather follow the format of the rest of the file and *not* have an empty line between each test case, fine by me!)